pig/src/main/java/org/apache/iceberg/pig/IcebergStorage.java

```diff
@@ -290,8 +265,8 @@ public RequiredFieldResponse pushProjection(RequiredFieldList requiredFieldList)
   }
 
   @Override
-  public void setUDFContextSignature(String signature) {
-    this.signature = signature;
+  public void setUDFContextSignature(String setSignature) {
```

Comment:
nit: instead of set or init, we've used new in other modules

Reply:
Thanks